resources/page/permalinks.go

```diff
@@ -173,8 +192,8 @@ func (l PermalinkExpander) validate(pp string) bool {
 		}
 
 		for _, match := range matches {
-			k := strings.ToLower(match[0][1:])
```

Comment:
Lower casing the key turns out to be a bug: If you use `:TITLE` as the key in the permalink configuration, it will pass this check, but then when trying to expand the permalink, it will fail because `TITLE` isn't in the map and the code would panic.

I decided to remove this to lower because it makes date formats like `Friday` not work because `friday` is not a valid Go time format.

I added a test for the behavior that non-normalized permalink configuration keys aren't valid. It would pass on master as well, due to the bug noted above.